core/server/proxy/src/main/java/alluxio/proxy/s3/ListBucketResult.java

```diff
@@ -0,0 +1,394 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.proxy.s3;
+
+import alluxio.AlluxioURI;
+import alluxio.client.file.URIStatus;
+import alluxio.wire.FileInfo;
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlElementWrapper;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Get bucket result defined in http://docs.aws.amazon.com/AmazonS3/latest/API/v2-RESTBucketGET.html
+ * It will be encoded into an XML string to be returned as an error response for the REST call.
+ *
+ * TODO(chaomin): consider add more required fields in S3 BucketGet API.
+ */
+@JacksonXmlRootElement(localName = "ListBucketResult")
+@JsonPropertyOrder({ "Name", "Prefix", "ContinuationToken", "NextContinuationToken",
+    "KeyCount", "MaxKeys", "IsTruncated", "Contents" })
+public class ListBucketResult {
+
+  // Name of the bucket.
+  private String mName = "";
+  // Keys that begin with the indicated prefix.
+  private String mPrefix = null;
+  // ContinuationToken is included in the response if it was sent with the request.
+  private String mContinuationToken = null;
+  // Returns the number of keys included in the response. The value is always less than or equal
+  // to the mMaxKeys value.
+  private int mKeyCount = 0;
+  // The maximum number of keys returned in the response body.
+  private int mMaxKeys = S3RestUtils.S3_DEFAULT_MAX_KEYS;
+  // Specifies whether or not all of the results were returned. If the number of
+  // results exceeds that specified by MaxKeys, all of the results might not be returned.
+  private boolean mIsTruncated = false;
+  // If only partial results are returned (i.e. mIsTruncated = true), this value is set as the
+  // next continuation token. The continuation token is the full Alluxio path of the object.
+  private String mNextContinuationToken = null;
+  // A list of objects with metadata.
+  private List<Content> mContents = new ArrayList<>();
```

Comment:
Do you mind changing the comment format to `/* */`?

Reply:
done